src/main/java/org/jenkinsci/test/acceptance/po/ConfigurablePageObject.java

```diff
@@ -126,8 +128,9 @@ public void ensureConfigPage() {
     public abstract URL getConfigUrl();
 
     public void save() {
-        clickButton("Save");
-        assertThat(driver, not(hasContent("This page expects a form submission")));
+        WebElement e = find(by.button("Save"));
+        e.click();
+        waitFor(e).until(CapybaraPortingLayerImpl::isStale);
```

Comment:
This line breaks my UI tests for Jenkins 2.263.x (it works with Jenkins 2.249.x):
- https://github.com/jenkinsci/warnings-ng-plugin/pull/758
- https://github.com/jenkinsci/git-forensics-plugin/pull/210